otelcol.yaml

```diff
@@ -0,0 +1,25 @@
+receivers:
```

Comment:
You probably didn't mean to commit this test file (and if you did, please don't put it in the root of the repo)

Reply:
Thanks, good catch.